contrib/completion/bash/docker

```diff
@@ -49,6 +49,9 @@
 # options immediately following their corresponding long form.
 # This order should be applied to lists, alternatives and code blocks.
 
+__docker_previous_extglob_setting=$(shopt -p extglob)
+shopt -s extglob
```

Comment:
Won't this only happen at script load?  Doesn't it need to happen directly in the functions which use the extended globbing since they'll be invoked at completion-time?